pages/program/abstracts/papers.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
I would say this probably doesn't need to be in the accordion format. Because there are only 4 papers. It was in accordion last year because there were a lot more and it would make the page _horrible_ to look at.